packages/actions/src/helpers/storage.ts

```diff
@@ -221,8 +221,8 @@ export const generatePreSignedUrlsPartsAPI = async (
   ceremonyId: number,
   token: string,
 ) => {
-  const url = new URL(`${process.env.API_URL}/storage/generate-pre-signed-urls-parts`);
-  url.search = new URLSearchParams({ ceremonyId: ceremonyId.toString() }).toString();
+  const url = new URL(`${process.env.API_URL}/storage/multipart/urls`);
+  url.search = new URLSearchParams({ id: ceremonyId.toString() }).toString();
```

Comment:
Same thing here, we need to add `userId` in the URL search params. Remember to follow the same steps as the previous comment